### ISSUE TITLE
Inconsistent indent in multi-line anonymous fn when zprint style is `indent-only`

### ISSUE THREAD

**Issue by tflavin** [kkinnear/zprint#243]
As written:
```clojure
#(let [really-long-text-so-it-puts-on-two-lines %]
         really-long-text-so-it-puts-on-two-lines)
```

Classic:
```clojure
#(let [really-long-text-so-it-puts-on-two-lines %]
   really-long-text-so-it-puts-on-two-lines)
```  

Indent only:
```clojure
#(let [really-long-text-so-it-puts-on-two-lines %]
  really-long-text-so-it-puts-on-two-lines)
```

**Comment by kkinnear** [comment]
Thanks for noticing this!  I have also fixed this, and it will be in the next release of zprint (almost certainly `1.2.4`).  I appreciate your attention to detail applied to `:indent-only`!  Thanks again!

**Comment by tflavin** [comment]
No problem - I first wanted to evaluate it across our codebase and initially thought it was just a difference in opinion from our current styling but then realized that it was inconsistent with classic. I think after `1.2.4` we'll be able to use `:indent-only`

**Comment by kkinnear** [comment]
Fixed in `1.2.4`, just out.
